latch/ldata/transfer/__init__.py

```diff
@@ -0,0 +1,3 @@
+from latch.ldata.transfer.download import download
```

Comment:
we do not put stuff in `__init__.py` because it makes importing anything take a million years

---

these renames are also technically breaking changes too because this stuff was public API for some reason ig lol